docs/reference/commandline/run.md

```diff
@@ -605,6 +605,34 @@ PS C:\> docker run --device=class/86E0D1E0-8089-11D0-9CE4-08003E301F73 mcr.micro
 > Windows containers. This option fails if the container isolation is `hyperv`
 > or when running Linux Containers on Windows (LCOW).
 
+### Access an NVIDIA GPU
+
+The `--gpus­` flag allows you to access NVIDIA GPU resources. Specify which GPUs
```

Comment:
Document dependency on nvidia-container-cli ?

Reply:
Well you've documented the dependency on the nvidia-container-toolkit package, which installs the CLI so you probably don't need to redocument this.